plugin/src/_rules/space-margin.js

```diff
@@ -0,0 +1,27 @@
+import { directionMap } from '#utils'
+
+export const spaceMargins = [
+  [/^space-([xy])-(\d+)$/, handlerSpace, { autocomplete: ['space-(x|y)', 'space-(x|y)-reverse', 'space-(x|y)-$spacing'] }],
+  [/^space-([xy])$/, handlerSpace],
+  [/^space-([xy])-reverse$/, ([, d]) => ({ [`--w-space-${d}-reverse`]: 1 })],
+]
+
+function handlerSpace([, d, s = 1], { theme }) {
+  const v = theme.spacing?.[s]
+  if (v != null) {
```

Comment:
Not sure if this makes sense, but im thinking that `theme.spacing` is an object and targeting not existing items within that object will return `undefined`, not `null`, right?

```suggestion
  if (!Boolean(v)) {
```

Reply:
Loose comparison to null catched both null and undefined

Reply:
Oki doki 😬 setting this to resolved